tests/test_common.py

```diff
@@ -140,6 +142,39 @@ def test_chroot_zip():
             assert b"data" == zip.read("directory/subdirectory/file")
 
 
+def test_chroot_zip_is_deterministic():
+    # type: () -> None
+    with temporary_dir() as tmp:
+        root_dir = os.path.join(tmp, "root")
+        os.mkdir(root_dir)
+        dir_a = os.path.join(root_dir, "a")
+        os.mkdir(dir_a)
```

Comment:
The `os.mkdir`s are fine, but touch takes care of this.

Reply:
Ah, forgot to follow up here. Looking into it now

Reply:
Done. Fingers crossed the new commit doesn't end up needing multiple CI runs.

Also thanks for the other review comments - I learned a few things.

Reply:
> Fingers crossed the new commit doesn't end up needing multiple CI runs.

The change in #2198, which just landed aheaded of this PR, should help with things going forward. If you continue to do development on Pex, you too can take advantage with `tox -e<test env> -- --devpi`.